File: src/androidTest/java/poetry/android/test/integration/java/data/models/Group.java
package poetry.android.test.integration.java.data.models;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

import poetry.annotations.MapFrom;

@DatabaseTable
public class Group
{
    @DatabaseField(id = true, columnName = "id")
	@MapFrom("id")
    private int id;

    @DatabaseField(columnName = "name")
	@MapFrom("name")
    private String name;

	public int getId()
	{
		return id;
	}

	public String getName()
	{
		return name;
	}
}

File: src/androidTest/java/poetry/android/test/integration/kotlin/models/Group.kt
package poetry.android.test.integration.kotlin.models

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable
import poetry.annotations.MapFrom

@DatabaseTable
class Group(
	@DatabaseField(id = true, columnName = "id")
	@MapFrom("id")
	var id: Int = 0,

	@DatabaseField(columnName = "name")
	@MapFrom("name")
	var name: String? = null
)
